tests/test_lane.py

```diff
@@ -3503,21 +3513,24 @@ def assert_contents(g, expect):
         _db = self._db
         eq_([], list(fiction.groups(self._db, facets=facets)))
 
-        # Here's an entry point that ignores everything except one
-        # specific book.
+        # Here's an entry point that applies a language filter
+        # that only finds one book.
         class LQRomanceEntryPoint(object):
             URI = ""
             @classmethod
-            def apply(cls, qu):
-                return qu.filter(work_model.sort_title=='LQ Romance')
-        facets = FeaturedFacets(0, entrypoint=LQRomanceEntryPoint)
```

Comment:
I had to change the way this works because you can't add a title restriction to our `Filter` object (though you can with an elasticsearch-dsl `F` object).